### ISSUE TITLE
Build rechunking tutorial

### ISSUE THREAD

**Issue by amsnyder** [hytest-org/hytest#277]
@rsignell-usgs is building a rechunking tutorial with the following structure:
(need to populate)

@gzt5142 is making sure the content populates a JupyterBook nicely. Current draft is here: https://gzt5142.github.io/DaskDataChunking/

Once the tutorial/book is complete, we will decide how to link to or incorporate it into the HyTEST JupyterBook.

**Comment by amsnyder** [comment]
@rsignell-usgs will be building the tutorial on NOAA GEFS retrospective data. The first two steps of his workflow (create individual file jsons and create consolidated metadata json/parquet file) will be redundant to what PUMP (@thompson) has already done, so Rich won't need to actually run this code - just run it on a sample of files to test and make sure it works.

Rich will actually build out the steps of the workflow to rechunk the data from the consolidated metadata file because PUMP has not worked on this part yet. We can provide the rechunked dataset to PUMP as a value-added substitution when it is ready.

**Comment by amsnyder** [comment]
Tasks:

- Infrastructure to build book is complete: https://github.com/hytest-org/DaskDataChunking
- Intro material in section 101 is essentially complete.
- @rsignell-usgs build out material for section 201 and update TOC to incorporate into book.
- @amsnyder will figure out how to pull into HyTEST Jupyterbook once complete.

**Comment by amsnyder** [comment]
closing - replaced by #534
